src/options/mkoptions.py

```diff
@@ -1437,9 +1448,9 @@ def mkoptions_main():
     # long options are available).
     for module in modules:
         for option in module.options:
-            check_links(option.filename, option.lineno, option.links)
+            check_links(option.filename, 0, option.links)
         for alias in module.aliases:
-            check_links(alias.filename, alias.lineno, alias.links)
+            check_links(alias.filename, 0, alias.links)
```

Comment:
I think the two calls to `check_links` above are the only ones. Should we maybe remove the `lineno` argument from the function?

Reply:
See comment below